exporter/opentelemetry-exporter-opencensus/src/opentelemetry/exporter/opencensus/metrics_exporter/__init__.py

```diff
@@ -121,13 +137,23 @@ def translate_to_collector(
             label_keys=label_keys,
         )
 
+        # If cumulative and stateful, explicitly set the start_timestamp to
+        # exporter start time.
+        if metric_record.instrument.meter.batcher.stateful:
```

Comment:
Maybe I am missing something from the previous comment, but is there also necessary a check for something being cumulative as well as being stateful?

Reply:
I believe if a metric is set to being stateful, then it is implied that it is a cumulative metric.

Reply:
That was my understanding, too